src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --search-big__height: 70px;
+  --search-small__height: 42px;
+}
+
+.search {
+  position: relative;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search__input {
+  width: 80%;
+  height: 100%;
+
+  font-family: "Avenir-Heavy", Arial , sans-serif;
+  font-weight: 900;
+  color: rgba(61, 78, 97, 1);
+
+  border: none;
+  outline: none;
+}
+
+.search-big {
+  height: var(--search-big__height);
+  margin-top: 20px;
+}
+
+.search-big__image {
+  position: absolute;
+  top: 24px;
+  left: 24.5px;
```

Comment:
/*this is not adequate, I spent a lot of time on it 0.5*/

Reply:
Could you use `left: 25px` ?

Reply:
I tried, but the tests don't pass